### ISSUE TITLE
[BUG] Unable to play TV episodes

### ISSUE THREAD

**Issue by saxoloman** [umbrellaplug/umbrellaplug.github.io#352]
### Bug

Greetings
for the past year or so i have this annoying issue where im unable to play tv episodes.
I select link i want to play and then nothing happens. no error message no popup , nothing...
So after few retries, sometimes 10 or more it eventually plays :D
Anyways if you feel like taking a look it is very much appreciated.
i have included logs

oh and it works fine when watching movies.

### Expected Result

1. select link after resolve
2. play tv episode
3. watch and eat some food
4. sleep :)

### Steps To Reproduce

1. find interesting tv show
2. resolve links
3. select links
4. oh no nothing happens
5. retry 

### Kodi Version, Type of Device, Logs, Screenshots, and Additional Info

maybe its a linux thing ?
works fine with windows,but i newer use windows for playback :(
i hope these logs shed some light as to whats wrong.
i am using slightly older version,but this happens on latest aswell.

[kodi.log](https://github.com/umbrellaplug/umbrellaplug.github.io/files/14961513/kodi.log)
[umbrella.log](https://github.com/umbrellaplug/umbrellaplug.github.io/files/14961515/umbrella.log)

if i can help in any other way,do tell.

### Checklist

- [X] I have included logs and the version of Kodi I am using..
- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have verified this is not a setting.

**Comment by umbrellaplug** [comment]
This has to do with the way I'm selecting the correct MyVideos.db.
From looking at the logs it appears I'm getting some type of failure in linux on detecting files in the directory to detect the correct database file to use. I will change the code to fix this.

**Comment by umbrellaplug** [comment]
Are you able to download the latest test version? I believe this issue is resolved, but I do not have a CoreElec setup to match yours to test.

**Comment by saxoloman** [comment]
Yea of course, thank you.
I made new install on usb for testing.
I tried test version but again its not working
[kodi.log](https://github.com/umbrellaplug/umbrellaplug.github.io/files/14963578/kodi.log)
[umbrella.log](https://github.com/umbrellaplug/umbrellaplug.github.io/files/14963579/umbrella.log)

**Comment by umbrellaplug** [comment]
Looking over this log the database issue previously noted is corrected. It appears the failure is coming from resolving from real-debrid.

```
2024-04-12 22:53:37.676 T:6113    error <general>: CCurlFile::Open - <https://95.download.real-debrid.com/d/I2UI232FWPBRM/Baby Reindeer S01E01 Episode 1 1080p NF WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv> Failed with code 0:
                                                   
2024-04-12 22:53:37.677 T:6113    error <general>: CCurlFile::CReadState::FillBuffer - (0xc5c4aee8) Failed: URL using bad/illegal format or missing URL(3)
2024-04-12 22:53:37.677 T:6113    error <general>: CCurlFile::Open - <https://95.download.real-debrid.com/d/I2UI232FWPBRM/Baby Reindeer S01E01 Episode 1 1080p NF WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv> Failed with code 0:
                                                   
2024-04-12 22:53:37.677 T:6113    error <general>: CFileCache::Open - <https://95.download.real-debrid.com/d/I2UI232FWPBRM/Baby Reindeer S01E01 Episode 1 1080p NF WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv> failed to open
```

I'm not sure if that's a general failure from real debrid or if it's caused by the file name. You may want to try a different source and see if it changes the results.

**Comment by umbrellaplug** [comment]
Also, can you make sure you have ipv6 disabled?

**Comment by saxoloman** [comment]
Ipv6 is disabled by default on coreelec. but i did double checked and its disabled.
also i tried changing my dns to 1.1.1.1 but its the same.
i also tried that link from ssh just to be sure and it works
CoreELEC:~ # wget https://95.download.real-debrid.com/d/I2UI232FWPBRM/Baby Reindeer S01E01 Ep
isode 1 1080p NF WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv
Connecting to 95.download.real-debrid.com (141.94.3.38:443)
saving to 'Baby'
Baby                   3% |*                                            | 63.0M  0:01:50 ETA^C
CoreELEC:~ #

**Comment by saxoloman** [comment]
i dont know much about python but normal curl from ssh also works
CoreELEC:~ # curl https://95.download.real-debrid.com/d/I2UI232FWPBRM/Baby Reindeer S01E01 Ep
isode 1 1080p NF WEB-DL DDP5 1 Atmos H 264-FLUX[EZTVx.to].mkv -o be.mkv
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                                     Dload   Upload  Total   Spent    Left     Speed
  6 1798M    6  119M    0     0     19.9M      0        0:01:30  0:00:06  0:01:24 22.5M   ^C
CoreELEC:~ #

if you do find amlogic device for coreelec you will see some weird stuff.
like 4k icons are not being displayed :)
but really,only issue is this one with resolving debri url :(

if you want i can install fen or some other addon and give you logs from them if they work

**Comment by saxoloman** [comment]
i reinstalled everything again.
when updating from older addon to new test version,i get playback problems.
when i install directly from test repo it works now.
i dont have playback problems on any tv shows that i tried.
sorry for confusions. seems it is resolved.

**Comment by umbrellaplug** [comment]
That's great news. Glad it's all working for you now. Did you use the same version of kodi on your fresh install?

**Comment by saxoloman** [comment]
yes, i used same version.
now i will have to reinstall everything on my main setup for this changes to take effect.
but finally no more playback problems :D
thank you
btw just a quick question. 
is there a way to use subscene as default search for subtitles instead of opensubtitles ?
i see subscene.py is still included just dont see a way to activate it from settings within addon.

**Comment by umbrellaplug** [comment]
Unfortunately not, I removed subscene as an option when the developer posted he was not sure he was going to be able to continue subscene. That file will be removed in the next release version. 

I hated to see it go also. Best bet these days is the new opensubs.com free level account allows 20 downloads a day for free. I've changed umbrella to use their new api since tie .org was retired.

**Comment by TwilightMercy** [comment]
> when the developer posted he was not sure he was going to be able to continue subscene.

Do you have a link for his post?

**Comment by umbrellaplug** [comment]
> if you do find amlogic device for coreelec you will see some weird stuff.
> like 4k icons in umbrella are not being displayed :)

Are the 4k icons missing still on your new install?

**Comment by saxoloman** [comment]
> > if you do find amlogic device for coreelec you will see some weird stuff.
> > like 4k icons in umbrella are not being displayed :)
> 
> Are the 4k icons missing still on your new install?

hey,sorry for late reply
yea they are still missing. I tried newest from test repo.
i made a picture :)
![MVIMG_20240416_192334](https://github.com/umbrellaplug/umbrellaplug.github.io/assets/99698419/851419c4-16ab-47a6-8b35-80c4f4db3349)

Also, i would like to ask you. If you may consider implementing secondary language option for subtitles (like it was once not so long ago in older umbrella version) . It is very much missed

thank you

**Comment by umbrellaplug** [comment]
I do not have a device to run CoreElec on unfortunately. I will see if I can find why those might be missing for you. I dealt with a similar issue in the past and it was due to the image names and case sensitivity on linux systems. I will check how those are named and verify.

On the second language for subs, I'll look into it. I believe I removed that due to the api changes with the new limits I was trying to minimize the amount of subtitles that were downloaded. If I remember correctly opensubs now allows 20 sub downloads a day.
